tslint.json

```diff
@@ -1,71 +1,20 @@
 {
+  "extends": "tslint-config-airbnb",
+  "defaultSeverity": "error",
   "rules": {
-    "ban": [false],
-    "class-name": true,
-    "comment-format": [false],
-    "curly": true,
-    "eofline": false,
-    "forin": true,
-    "indent": [true, 4],
-    "interface-name": false,
-    "jsdoc-format": false,
-    "label-position": true,
-    "label-undefined": true,
-    "member-ordering": [true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
+    "no-debugger": 2,
+    "no-console": 1,
+    "prefer-array-literal": [ true, { "allow-type-parameters": true } ],
+    "variable-name": [ true, "allow-pascal-case", "allow-leading-underscore" ],
```

Comment:
allow-leading-underscore should be removed.

Reply:
leading underscores used to be part of the MediaMonks coding standards, did we decide on dropping this?